### DISCUSSION TITLE
Sort styles

### DISCUSSION THREAD

**Discussion by mikuhl-dev** [chakra-ui/panda#875] [Ideas]
### Description

I would love to see Panda CSS sort styles, like Tailwind does.

### Problem Statement/Justification

This makes your styles more readable as they will always be in a predictable order.

### Proposed Solution or API

Maybe a Prettier plugin?

**Comment by astahmer** [comment]
I haven't looked into it but I wonder if it could be done with https://github.com/azat-io/eslint-plugin-perfectionist
